tests/testrom.c

```diff
@@ -189,22 +189,23 @@ static const struct Testsuite
 	uint32_t duration;
 	uint32_t flags;
 } tests[] = {
-	TEST_FUNC(test_exception,              	   5, TEST_FLAGS_NO_BENCHMARK),
```

Comment:
This block was a mix of tabs and spaces, which was causing alignment issues. I have made it consistent regardless of tab size by using a leading tab (no change), and spaces for field alignment padding.